clickhouse_mysql/writer/chwriter.py

```diff
@@ -68,15 +68,25 @@ def insert(self, event_or_events=None):
         event_converted = None
         for event in events:
             if not event.verify:
-                logging.warning('Event verification failed. Skip one event. Event: %s Class: %s', event.meta(), __class__)
-                continue # for event
+                logging.warning(
+                    'Event verification failed. Skip one event. Event: %s Class: %s', event.meta(), __class__)
+                continue  # for event
 
             event_converted = self.convert(event)
             for row in event_converted:
+                # These columns are added to identify the last change (tb_upd) and the kind of operation performed
+                # 0 - INSERT, 1 - UPDATE, 2 - DELETE
+                row['tb_upd'] = datetime.datetime.now()
+                row['operation'] = 0
+
                 for key in row.keys():
-                    # we need to convert Decimal value to str value for suitable for table structure
-                    if type(row[key]) == Decimal:
+                    # we need to convert Decimal or timedelta value to str value for suitable for table structure
+                    if type(row[key]) == [Decimal, datetime.timedelta]:
```

Comment:
Should this be `in`?
```suggestion
                    if type(row[key]) in [Decimal, datetime.timedelta]:
```